cypress/e2e/swap.test.ts

```diff
@@ -135,28 +135,43 @@ describe('Swap', () => {
             cy.get('#swap-button').click()
             cy.get('#confirm-swap-or-send').click()
 
-            // Mine a block past the deadline
-            cy.then(() => hardhat.provider.send('eth_getBlockByNumber', ['latest']))
+            // Mine a block past the deadline.
+            cy.then(() => hardhat.provider.send('eth_getBlockByNumber', ['latest', false]))
               .then((block) => block.timestamp * 2)
               .then((timestampIncrease) => {
                 hardhat.provider.send('evm_increaseTime', [hexValue(timestampIncrease)])
                 hardhat.provider.send('hardhat_mine', [hexValue(1), hexValue(timestampIncrease)])
               })
```

Comment:
Why not use `hardhat.mine`? Why do you need to predicate this off of the data from `eth_getBlockByNumber`?